app-frontend/src/app/index.routes.js

```diff
@@ -20,10 +20,6 @@ import projectsTpl from './pages/projects/projects.html';
 import projectsNavbarTpl from './pages/projects/navbar/navbar.html';
 import projectsEditTpl from './pages/projects/edit/edit.html';
 
-import projectsEditColorTpl from './pages/projects/edit/color/color.html';
-import projectsEditColormodeTpl from './pages/projects/edit/colormode/colormode.html';
-import projectsAdvancedColorTpl from './pages/projects/edit/advancedcolor/advancedcolor.html';
-import projectsColorAdjustTpl from './pages/projects/edit/advancedcolor/adjust/adjust.html';
```

Comment:
Is the only change removing the relevant routes for these templates?
I'm thinking we should add a format command like annotate has, and run it once on all files to set a new baseline since this is a a lot of noise

Reply:
Yes that's correct. The bigger change is probably from setting up my editor to autoformat specifically because of annotate. I'm happy to hit the whole `app-frontend` dir with a `yarn format` as part of this PR. Do you think that makes sense?

Reply:
yup. I don't think we have a "format" command in RF, but I'm good with adding one and running it to clean things up going forward

Reply:
I added a `format` script -- we already had a `.prettierrc`. There's no ci check but at least we'll all be starting from the same place now.